.github/workflows/wheels.yml

```diff
@@ -19,83 +19,32 @@ on:
 
 jobs:
   make_sdist:
-    name: "SDist: ${{ matrix.os }}"
+    name: SDist
     if: ${{ github.event_name != 'pull_request' || !github.event.pull_request.draft }}
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
+    runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
 
       - name: Setup Python
-        uses: actions/setup-python@v3.1.2
+        uses: actions/setup-python@v3
 
       - name: Make sdist
         run: |
-          python --version
-          pipx --version
-          pipx run build --sdist
-
-      - name: Install from sdist
-        run: |
-          pip install --upgrade pip setuptools wheel
-          pip install pytest
-          cp dist/h3-*.tar.gz h3.tar.gz
-          pip install h3.tar.gz[all]
-
-      - name: Test sdist
-        run: pytest --cov=h3 --full-trace
+          pip install --upgrade pip setuptools
+          pip install --upgrade build
```

Comment:
Install on one line so that pip can resolve dependencies jointly?

Reply:
My habit has been to upgrade pip and setuptools first, to ensure that we have the latest version of pip that actually includes the (fairly new) dependency resolver. Not sure if that's entirely necessary here, so I just have it this way to be safe. Also, to ensure that we're installing `build` using the latest version of `pip`.

And I don't think we'd have any dependency issues between these three packages anyway. At least, if there were, that'd be something we'd want to pin explicitly. Would you agree?

Reply:
Interesting. That's never crossed my mind before. I've just defaulted to installing on one line, but makes sense that might use the resolver of the original pip version, not the new one. Seems fine to leave as is